wordpress-mobile/WordPress-iOS
# Issue/5849 Update Editor nav bar buttons

Fixes #5849. The nav bar in the editor now has its right bar button options under a 'more' menu (and an Edit button when appropriate). This makes things look way less cluttered. I also took the liberty of removing the Edit button tooltip, as it was a hack to position in the right place and it was kind of unnecessary (Tap Edit to Edit, huh?).

<img width="375" alt="screen shot 2016-08-11 at 15 57 55" src="https://cloud.githubusercontent.com/assets/4780/17593039/7258cf8c-5fdc-11e6-9e78-d42c77615f0e.png">

To test:

* Ensure that the More button behaves as you'd expect (on both Posts and Pages, when entering / exiting edit mode...) and shows the right options. 
* The Save / Update / Post button should appear next to the More button in edit mode, should be enabled when there are changes, and show the correct title.
* Ensure that each option in the menu works correctly.

Needs review: @aerych 
Would you do the honours? Thanks!
